src/hdf5_hl.ts

```diff
@@ -106,11 +110,12 @@ function process_data(data: Uint8Array, metadata: Metadata, json_compatible: boo
   // but otherwise returns Uint8Array raw bytes as loaded.
   let output_data: OutputData;
   let { shape, type } = metadata;
+
   let known_type = true;
   // let length: number;
   if (type === Module.H5T_class_t.H5T_STRING.value) {
     if (metadata.vlen) {
-      let output = [];
+      let output: string[] = [];
```

Comment:
The more recent version of TypeScript in VS Code infers this as `never[]` and shows an error on the line with `output.push()` below. There were a few cases like this.